src/test/ui/issues/issue-61108.stderr

```diff
@@ -6,11 +6,17 @@ LL |     let mut bad_letters = vec!['e', 't', 'o', 'i'];
 LL |     for l in bad_letters {
    |              -----------
    |              |
-   |              value moved here
+   |              value moved due to this method call
    |              help: consider borrowing to avoid moving into the for loop: `&bad_letters`
 ...
 LL |     bad_letters.push('s');
    |     ^^^^^^^^^^^ value borrowed here after move
+   |
+note: this function takes `self`, which moves the receiver
```

Comment:
I wonder if it'd make sense to change the wording here slightly to something like (wording very much up to discussion)

    this function consumes the receiver `self` by taking ownership of it, which moves `bad_letters`

"receiver" is jargon that we don't want to completely hide (lets people google for the concept) but we want to make it unambiguous that `bad_letters` is being moved because `self` is being "consumed"/moved.